frontend/src/App.tsx

```diff
@@ -339,6 +341,16 @@ export class App extends PureComponent<Props, State> {
     })
 
     this.handleSessionStateChanged(sessionState)
+
+    // send current URL to server
+    this.sendCurrentUrlToServer()
+  }
+
+  // This is only used in st.experimental.get_query_string
```

Comment:
This message is sent in any case, right? Whether we are using the `st.get/set_query...` or not.

Reply:
Yes, I will modify the comment to make it more proper.